src/Nest/Aggregations/AggregateJsonConverter.cs

```diff
@@ -10,580 +10,147 @@ namespace Nest
 {
 	internal class AggregateJsonConverter : JsonConverter
 	{
-		private static readonly Regex _numeric = new Regex(@"^[\d.]+(\.[\d.]+)?$");
-
-		public override bool CanWrite => false;
-
-		public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer) => this.ReadAggregate(reader, serializer);
+		private static Dictionary<string, Type> _aggregateTypes = new Dictionary<string, Type>
```

Comment:
Can we write a CodeStandards tests that reflects this dict for all know aggregation types?